src/ReadingBuddy.java

```diff
@@ -1101,6 +1230,18 @@ public static ArrayList<SearchPost> searchForSearchPosts(String title, String au
         return matches;
     }
 
+    //NEW SPRINT3: (misc--ability to find a specific searchpost (connected to a specific user))
```

Comment:
this goes along with a miscellaneous update earlier in the code (not actually a part of this ending a group feature, just an error noticed). When a CurrentBook is being removed, if its hasOpenSearch attribute is true, this function enables the deletion of the accompanying SearchPost.